spec/models/labware_creators/quadrant_stamp_primer_panel_spec.rb

```diff
@@ -304,7 +305,8 @@
           'custom_metadatum_collection-uuid',
           body: json(:v1_custom_metadatum_collection, uuid: 'custom_metadatum_collection-uuid')
         )
-        stub_api_get('user-uuid', body: user)
+        stub_api_get('user-uuid', body: v1_user)
```

Comment:
Not really, no.  Removing the line does make the test fail though because the endpoint does get queried:

```
WebMock::NetConnectNotAllowedError:
       Real HTTP connections are disabled. Unregistered request: GET http://example.com:3000/user-uuid

     # ./app/models/labware_metadata.rb:21:in `update!'
     # ./app/models/labware_creators/quadrant_stamp_base.rb:39:in `block in create_labware!'
     # ./app/models/labware_creators/multi_stamp.rb:40:in `create_labware!'
     # ./app/models/labware_creators/quadrant_stamp_base.rb:38:in `create_labware!'
     # ./app/models/labware_creators/base.rb:56:in `save'
     # ./app/models/labware_creators/base.rb:52:in `save!'
     # ./spec/models/labware_creators/quadrant_stamp_primer_panel_spec.rb:328:in `block (4 levels) in <top (required)>'
```